docs/docs/using-unstructured-data.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Using Unstructured Data
+---
+
+This is a stub. Help our community expand it.
+
+Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
+pull request gets accepted.
+
+Initial notes:
+[GitHub Issue #8482](https://github.com/gatsbyjs/gatsby/issues/8482)
+
+### Do I have to use GraphQL and source plugins to pull data into Gatsby sites?
+
+Absolutely not! You can use the node `createPages` API to pull unstructured data into Gatsby sites rather than GraphQL and source plugins. This is a great choice for small sites, while GraphQL and source plugins can help save time with more complex sites.
+
+### Prerequisites (if any)
+
+### The facts
+
+### Example
+
+**to do** Possibly pull from these examples
+https://twitter.com/thekitze/status/1043532445742784515
```

Comment:
This is an example of the type of observation we're trying to explain toward -- imo relevant for context in the issue, not to be in the doc, I don't think.